xocto/ranges.py

```diff
@@ -1085,34 +1115,3 @@ def iterate_over_months(
 
         yield FiniteDatetimeRange(start_at, this_end)
         start_at = next_start
-
-
-def date_range_for_midnight_range(
```

Comment:
If you want to avoid another major version / breaking change, consider changing the body of this to call the method?

Reply:
I'd prefer to break: it's recent, so usages will very low if any (non in Kraken Core). Going through a full deprecation process doesn't seem worth it - but we can always change approach if someone complains.